src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IProtocolHandler.kt

```diff
@@ -74,6 +74,8 @@ interface IProtocolHandler : AutoCloseable {
      */
     fun onOutgoing(message: ByteBuf, metadata: MutableMap<String, String>) = Unit
 
+    fun getGrpcService()
```

Comment:
It should take service class as a parameter and return service instance. Also why it is in this change?